style.css

```diff
@@ -98,3 +101,343 @@ body {
+    margin-top: 5px;
+    margin-right: 12px;
+  }
+
+  .card-description {
+    width: 295px;
+    height: 76px;
+    font-style: normal;
+    font-weight: 400;
+    font-size: 15px;
+    line-height: 24px;
+    color: #344563;
+    margin-top: 20px;
+  }
+
+  .card-list {
+    display: flex;
+    list-style-type: none;
+    margin-bottom: 12px;
+  }
+
+  .card-list-tags-html {
+    color: #6070ff;
+    width: 53px;
+    height: 24px;
+    background: #ebebff;
+    border-radius: 8px;
+    text-align: center;
+    padding: 4px 12px;
+    font-style: normal;
+    font-weight: 500;
+    font-size: 12px;
+    margin-top: 12px;
+    margin-right: 8px;
+  }
+
+  .card-list-tags-css {
+    color: #6070ff;
+    width: 53px;
+    height: 24px;
+    background: #ebebff;
+    border-radius: 8px;
+    text-align: center;
+    padding: 4px 12px;
+    font-style: normal;
+    font-weight: 500;
+    font-size: 12px;
+    margin-top: 12px;
+    margin-right: 8px;
+  }
+
+  .card-list-tags-js {
+    color: #6070ff;
+    width: 89px;
+    height: 24px;
+    background: #ebebff;
+    border-radius: 8px;
+    text-align: center;
+    padding: 4px 12px;
+    font-style: normal;
+    font-weight: 500;
+    font-size: 12px;
+    margin-top: 12px;
+    margin-bottom: 24px;
+  }
+
+  .card-link {
+    align-items: center;
+    text-decoration: none;
+    gap: 10px;
+    width: 124px;
+    height: 48px;
+    border: 1px solid #6070ff;
+    border-radius: 8px;
+    margin-bottom: 28px;
+    padding: 12px;
+  }
+
+  .second-design {
+    background-color: #fff;
+    margin-top: 104px;
+    margin-bottom: 24px;
+    margin-right: 24px;
+    margin-left: 24px;
+    border-radius: 20px;
+    width: 327px;
+    height: 552px;
+    border: 1px solid #dfe1e6;
+  }
+
+  .third-design {
+    background-color: #fff;
+    margin-top: 104px;
+    margin-bottom: 24px;
+    margin-right: 24px;
+    margin-left: 24px;
+    border-radius: 20px;
+    width: 327px;
+    height: 552px;
+    border: 1px solid #dfe1e6;
+  }
+
+  .fourth-design {
+    background-color: #fff;
+    margin-top: 104px;
+    margin-bottom: 24px;
+    margin-right: 24px;
+    margin-left: 24px;
+    border-radius: 20px;
+    width: 327px;
+    height: 552px;
+    border: 1px solid #dfe1e6;
+  }
+
+  .top-section {
+    background-position: center;
+    background-repeat: no-repeat;
+    display: flex;
+    flex-direction: column;
+    align-items: flex-start;
+    padding: 120px 24px;
+    gap: 12px;
+    border-top-right-radius: 60px;
+    background-color: #fff;
+  }
+
+  .top-title {
+    font-family: 'Poppins', sans-serif;
+    font-style: normal;
+    font-weight: 700;
+    font-size: 40px;
+    line-height: 52px;
+    width: 327px;
+    margin-top: -50px;
+    color: #172b4d;
+  }
+
+  .btn-link {
+    align-items: center;
+    text-decoration: none;
+    padding: 12px;
+    gap: 10px;
+    width: 124px;
+    height: 48px;
+    border: 1px solid #6070ff;
+    border-radius: 8px;
+  }
```

Comment:
- [ ] Consider adding a hover and pressed state to your buttons to match the figma design. You can use the `:hover` and `:active` pseudo-classes.
![btn1](https://user-images.githubusercontent.com/94551484/198816167-a455bb39-71de-425b-bb21-611d340ad82d.png)

Reply:
issue resolved. Thank you